spec/compiler/normalize/expressions_spec.cr

```diff
@@ -0,0 +1,15 @@
+require "../../spec_helper"
+
+describe "Normalize: expressions" do
+  it "normalizes an empty expression" do
+    assert_normalize "begin\nend", ""
+  end
+
+  it "normalizes an expression" do
+    assert_normalize "(1 < 2).as(Bool)", "((1 < 2)).as(Bool)"
```

Comment:
It is `ASTNode#to_s` bug, but it is not related to this PR, and I've fixed it in my local repository already. I'll open a new PR.